cockroachdb/templates/job.init.yaml

```diff
@@ -20,7 +20,11 @@ metadata:
   {{- end }}
   annotations:
     helm.sh/hook: post-install,post-upgrade
+    {{- if .Values.init.deleteJobOnCompletion }}
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    {{- else }}
     helm.sh/hook-delete-policy: before-hook-creation
+    {{- end }}
```

Comment:
We can keep this deletion behaviour as default one and add an option in values.yaml to disable this